src/v2/view-builder/components/AuthenticatorEnrollOptions.js

```diff
@@ -42,7 +44,33 @@ const AuthenticatorRow = View.extend({
   },
   minimize: function() {
     this.$el.addClass('authenticator-row-min');
-  }
+  },
+  getTemplateData() {
+    let authenticatorUsageText;
+    const allowedFor = this.model.get('relatesTo').allowedFor;
+
+    if (allowedFor) {
+      switch (allowedFor) {
+      case 'any':
```

Comment:
make them const ?

Reply:
is this the right spot to put such constants `src/v2/view-builder/utils/Constants.js`

Reply:
yeah.. why not ?